microsoft-edge/dualengine/get-started.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Getting Started with DualEngine API
+description: Get started using DualEngine API in your application.
+author: MSEdgeTeam
+ms.author: edgededev
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.localizationpriority: high
+ms.technology: windows-integration
+ms.date: 11/06/2023
+---
+# Getting Started with DualEngine API
+In this article we will walk you through the steps needed to start using the DualEngine API.
+
+## Step 1 - Receiving Access to the DualEngine LAF
+Access to the DualEngine API is controlled via a Limited Access Feature.
+To gain access to the feature reach out to dualengineapiaccess@microsoft.com
+
+Those familiar with LAF should note that the DualEngine LAF is handled somewhat differently than is typical, the 
+details of which can be found in the [Limited Access Feature Requirements](concepts/adapter-dll.md#limited-access-feature-requirements)
+
+## Step 2 - Download the DualEngine API header
+The DualEngine API is a COM interface that we only officially support via C++ and is not part of the normal Windows
+SDK. As such we have provided a header that you must include in your adapter DLL project which can be downloaded from
+[DualEngine API Header Download]().

```

Comment:
@mikehoffms Is there guidance for how to distribute something like a C/C++ header like this?

Reply:
@captainbrosset - Is there guidance for how to distribute something like a C/C++ header like this?

Reply:
Contrib Guide > Where to host downloadable files
https://review.learn.microsoft.com/en-us/help/contribute/downloadable-files?branch=main
@AndrewLakeMSFT @captainbrosset

Reply:
fresh Files Changed tab shows that this link is still TBD

Reply:
fresh Files Changed tab shows:
```
Therefore, you must include the following header in your adapter DLL project: 
[DualEngine API Header Download](https://download.microsoft.com/download/c/5/0/c5035487-bd78-4fd0-9cc4-e1c5a3b654b7/dualengine.h)
```